app/assets/stylesheets/wizard.scss

```
@@ -607,15 +597,14 @@ body.wizard {
     .help-text {
       margin: 2em 0;
     }
-
-    .primary {
-      display: inline-block;
-    }
   }
 }
 
-// Register admin page... /finish-installation/register
+// Register admin pages
+// /finish-installation and /finish-installation/register
 .wizard-container-contents.finish-installation {
+  font-family: system-ui, Arial, sans-serif;
```

can we use `--font-family` here?
```suggestion
  font-family: var(--font-family);
```